src/algorithms/sequential_algorithm.jl

```diff
@@ -19,7 +19,17 @@ function build_main_problem!(
     container::MultiOptimizationContainer{SequentialAlgorithm},
     template::MultiProblemTemplate,
     sys::PSY.System,
-) end
+)
+    for k in keys(container.subproblems)
```

Comment:
Why is this needed? The isolated bus thing should be figured out from the data.

Reply:
Jus the isolated part? or the mapping in general? The mapping is for knowing which subproblem should updated the active power balance expression for the buses.

Reply:
Just the isolated filter